package.json

```diff
@@ -55,10 +55,13 @@
     "heroku-debug-5.0.2": "file:test/heroku-debug/5.0.2",
     "heroku-debug-5.0.3": "file:test/heroku-debug/5.0.3",
     "heroku-hello-world-build": "0.0.0",
+    "heroku-kafka": "2.9.8",
```

Comment:
@dickeyxxx the integration tests install `heroku-pg-extras` and `heroku-kafka` so that I can verify that they are able to co-exists, adding them to dev dependencies speeds things up because then they exist in the offline cache